ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -113,53 +82,58 @@ message CreatedEvent {
   ContractMetadata metadata = 10;
 }
 
-// View of a create event matched by an interface filter.
-message InterfaceView {
+// Records that a contract has been transferred-out, and it becomes unusable on the source domain
+message TransferredOutEvent {
```

Comment:
What about indicating whether the participant on whose completion stream this event is observed is actually able to submit a transfer-in? Only participants that are declared as transferring can currently initiate transfer-in event. But non-transferring transfer-out participants should also emit such an event.

One option is to encode this information in the transfer-in exclusivity: It's set to infinity on non-transferring transfer-out participants.